src/api/Flooq.Api/Controllers/DataFlowController.cs

```diff
@@ -0,0 +1,120 @@
+#nullable disable
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Flooq.Api.Domain;
+using Flooq.Api.Model;
```

Comment:
Seems that you have some ambiguous using directives causing the build to fail. I think you do not need System, System.Threading.Tasks and System.Collections.Generic. Try removing these and then we will see whether the build still fails, but I think that might be the hint.

Reply:
Thanks for pointing this out. I removed them and it works.